app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -558,16 +629,16 @@ private void loadItems() {
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(
-                    result -> {
-                        feed = result;
-                        refreshHeaderView();
-                        displayList();
-                    }, error -> {
-                        feed = null;
-                        refreshHeaderView();
-                        displayList();
-                        Log.e(TAG, Log.getStackTraceString(error));
-                    });
+                        result -> {
+                            feed = result;
+                            refreshHeaderView();
+                            displayList();
+                        }, error -> {
+                            feed = null;
+                            refreshHeaderView();
+                            displayList();
+                            Log.e(TAG, Log.getStackTraceString(error));
+                        });
```

Comment:
If you completely revert the unrelated whitespace formatting, Checkstyle should not complain. It only complains about lines that you changed.